### DISCUSSION TITLE
Configure metadata on MySQL using TLS

### DISCUSSION THREAD

**Discussion by solracsf** [juicedata/juicefs#2217] [Q&A] [answered]
There are instruction for Redis, PostgreSQL and TiKV using SSL/TLS, but nothing about MySQL/MariaDB.
Can this be added to https://juicefs.com/docs/community/databases_for_metadata/#mysql ?

**Comment by xiaogaozi** [comment, marked as answer]
You could add [`tls=true`](https://github.com/go-sql-driver/mysql#tls) to the metadata URL to enable TLS feature. Contributions to documentation are very welcome.
